### DISCUSSION TITLE
EasyJet Specific Announcements - British Accent

### DISCUSSION THREAD

**Discussion by leontgiscombe** [brian-mckeown/flightdash#92] [General]
Hi, is it possible to have British only accents? I believe the fable accent is the chosen British accent. If I select fable for each cabin crew will they all sound the same or slightly different? Thought I would ask rather than waste some cash checking each sound pack on the web player. Thanks!

**Comment by brian-mckeown** [comment]
Hey, thanks for the question! Currently, we use OpenAI's Text To Speech model for generating the voices. At this time, they only have the 6 voice options available. And you are correct, fable is the only one with a British accent. If you select fable for each, it would indeed sound like the exact same person speaking in each announcement, though what they are saying would be different. 
We do hope OpenAI will expand their voice options in the future. We may also explore other Text To Speech options in the AI market in the future. 
Here is a link to listen to their current options (for others that may stumble upon this post): https://platform.openai.com/docs/guides/text-to-speech

**Reply by leontgiscombe** [reply]
Hi Brian,

Really appreciate the speedy and well informative response. This won’t deter me from using the announcements as the immersion is next to none. Really looking forward for the advancement of these open ai text to speech and what is to come. Keep up the awesome work for the sim community(your EFB is very very useful, really glad I stumbled across it, especially the vatmap) Kudos.
